test/remote/gateways/remote_beanstream_test.rb

```diff
@@ -384,6 +384,7 @@ def test_transcript_scrubbing
     assert_scrubbed(@visa.number, clean_transcript)
     assert_scrubbed(@visa.verification_value.to_s, clean_transcript)
     assert_scrubbed(@gateway.options[:password], clean_transcript)
+    assert_scrubbed(@gateway.options[:api_key], clean_transcript)
```

Comment:
Should probably also be added to the example transcript in the scrubbing unit test

Reply:
It actually is not present in the transcript. I am more so thinking of this as a precaution.

Reply:
Gotcha 👍